joint_groups:
  test_group:
    sync_read:
      - position
    sync_write:
      - position

joint1: { id : 0, operating_mode: 3}
